src/core/messages.ts

```diff
@@ -496,9 +496,10 @@ export class DefaultMessages
       text,
       metadata ?? {},
       headers ?? {},
-      new Date(response.createdAt),
       ChatMessageActions.MessageCreate,
       response.serial,
+      new Date(response.createdAt),
```

Comment:
Maybe worth just adding a small comment to say, timestamp and createdAt are equal, or doing above const timestamp = response.createdAt so its explicit that this logic was wanted?

Reply:
added comment